index.rst

```diff
@@ -1,61 +1,504 @@
+This is already the case at the IDF.
+
+- *[REQ-WFL-003] Failed quanta must be reported including where in the batch processing system the quantum was running at the time of failure.*
+
+This will be satisfied by implementing the design described in [:ref:`provenance-schema`] and [:ref:`recording-provenance`], provided the ``host`` field in the ``quantum`` table is consistent with the "where" question here.
+
+- *[REQ-WFL-004] Though no requirement exists, it should be possible to inspect, post-facto, the resource usage (CPU, memory, I/O etc.) for individual workers.*
+
+This is already implemented for provenance queries that start with the data ID of the quantum to be inspected, because these values are stored in the task metadata dataset (at a coarse level by the execution system, and optionally with more fine-grained information by the task itself).
+The rest of the design described in this document should allow this information to be connected with worker nodes.
+
+Observatory-Butler linkage
+--------------------------
+
+Possibly relevant recommendations:
+
+- *[REC-EXP-1] As planned, program details known to the scheduler (such as science programme and campaign name) should be captured by the Butler.*
+
+As long as these details are included in the FITS headers of the raw files, they can be configured to be capture by the butler and stored in the ``exposure`` dimension table.
+This is already the case for the fields used as examples in this recommendation.
+
+- *[REC-EXP-2] As planned, OCS queue submissions that result in meaningfully grouped observations should be identified as such in the Butler.*
+
+This has been implemented by :jira:`DM-33602` and :jira:`DM-30948`.
+
+Metrics linkage
+---------------
+
+- *[REC-MET-001] For metrics that can be associated with a Butler dataId, the metrics should be persisted using the Data Butler as the source of truth. The dataId associated with the metric should use the full granularity.*
+- *[REC-MET-002] Any system that uses Butler data to derive metrics should persist them in the Butler provided that the metrics are associable with a Data ID.*
+
+This is already implemented and in regular use in the ``faro`` and ``analysis_tools`` packages, and is currently the only way that metrics derived from butler data are initially stored.
+It is arguable whether the butler datasets are considered the source of truth after upload to SQuaSH (:cite:`SQR-008`).
+:cite:`DMTN-203` will explore these details more fully.
+
+- *[REC-MET-003] When lsst.verify.Job objects are exported, the exported object should include the needed information (run collection and dataId) to associate with the source of truth metric persisted with the Data Butler.*
+
+At prsent it is unusual for butler datasets to store their own data ID and especially their own ``RUN`` collection internally, but this is something expect to do more in the future (see discussion on *REC-FIL-1* below).
```

Comment:
typo: prsent

"this is something expect to do more" does not parse :slightly_smiling_face: